api/v1/cluster_types.go

```diff
@@ -309,6 +309,14 @@ type ClusterSpec struct {
 	// The SeccompProfile applied to every Pod and Container.
 	// Defaults to: `RuntimeDefault`
 	SeccompProfile *corev1.SeccompProfile `json:"seccompProfile,omitempty"`
+
+	// ReplicaLagThreshold is a user-customizable field that represents the acceptable lag limit (in megabytes,
+	// where 1MB = 1,000,000 Bytes) between the primary database and its replicas. If a replica's lag exceeds this
```

Comment:
ReplicaLagThreshold is a user-customizable field that represents the acceptable lag limit (in megabytes... )
========
The default value of the ReplicaLagThreshold is 30,000,000, which is in Bytes, not in megabytes. we should either change the descrption or change the unit used for ReplicaLagThreshold